src/statscollector/DefaultStatsCollector.ts

```diff
@@ -420,6 +420,11 @@ export default class DefaultStatsCollector implements StatsCollector {
   }
 
   private handleRawMetricReports(rawMetricReports: RawMetricReport[]): void {
```

Comment:
Although we are calling this param `rawMetricReports`, it's not untouched data from `rtcPeerConnection.getStats`.

From 453-460, SDK is re-assigning raw data to a specific object shape.
https://github.com/aws/amazon-chime-sdk-js/blob/f8c345b95472cc2aea26e7e399ca72f3523900d2/src/statscollector/DefaultStatsCollector.ts#L453

I wonder if a builder wants raw data directly from `rtcPeerConnection.getStats()`.

Reply:
Good point. Updated.